src/plugins/postgres_table_copy_operator.py

```diff
@@ -112,6 +117,54 @@ def __init__(
                 "even though source table will be dropped."
             )
 
+    def _drop_tables_if_unequal(self, cursor: Any, all_table_copies: List[TableMapping]) -> None:
+        """Drop the target tables that are unequal to their source table."""
+        for table_mapping in all_table_copies:
+            # First make sure the target table exists
+            cursor.execute(
+                sql.SQL("SELECT to_regclass('{target_table_name}')").format(
+                    target_table_name=sql.Identifier(table_mapping.target)
+                )
+            )
+
+            if first(cursor.fetchone()) is None:
+                self.log.info(
+                    "Target table %s not found, skipping equality check.", table_mapping.target
+                )
+                continue
+            cursor.execute(
+                sql.SQL(
```

Comment:
No need for `sql.SQL` here. You are not doing anything fance with `sql.Identifier` or `sql.Placeholder` or whatever. This is just a regular SQL query with DBAPI parameters (the `%(...)s ones).

BTW I like this query. Nice solution.